runtime/moonbase/src/lib.rs

```diff
@@ -471,29 +475,25 @@ parameter_types! {
 }
 
 impl pallet_scheduler::Config for Runtime {
-	type Event = Event;
-	type Origin = Origin;
+	type RuntimeEvent = RuntimeEvent;
+	type RuntimeOrigin = RuntimeOrigin;
 	type PalletsOrigin = OriginCaller;
-	type Call = Call;
+	type RuntimeCall = RuntimeCall;
 	type MaximumWeight = MaximumSchedulerWeight;
 	type ScheduleOrigin = EnsureRoot<AccountId>;
 	type MaxScheduledPerBlock = ConstU32<50>;
 	type WeightInfo = pallet_scheduler::weights::SubstrateWeight<Runtime>;
 	type OriginPrivilegeCmp = EqualPrivilegeOnly;
-	// Preimage provider with which we look up call hashes to get the call
-	type PreimageProvider = Preimage;
-	// Number of blocks to postpone execution for when the item is delayed
-	type NoPreimagePostponement = NoPreimagePostponement;
+	type Preimages = Preimage;
 }
 
 impl pallet_preimage::Config for Runtime {
 	type WeightInfo = pallet_preimage::weights::SubstrateWeight<Runtime>;
-	type Event = Event;
+	type RuntimeEvent = RuntimeEvent;
 	type Currency = Balances;
 	type ManagerOrigin = EnsureRoot<AccountId>;
-	type MaxSize = ConstU32<{ 4096 * 1024 }>;
-	type BaseDeposit = ConstU128<{ 5 * currency::UNIT * currency::SUPPLY_FACTOR }>;
-	type ByteDeposit = ConstU128<{ 1 * currency::UNIT * currency::SUPPLY_FACTOR }>;
+	type BaseDeposit = ConstU128<{ 0 * currency::UNIT * currency::SUPPLY_FACTOR }>;
```

Comment:
Why 0?

Reply:
I did set the same value that we had for preimage deposits in pallet-democracy. In pallet-democracy, we didnt have a base Deposit, we only had a byteDeposit, hence the reason for making it 0. We can change it if we dont feel comfortable not charging a baseDeposit

Reply:
A base deposit might be important to prevent any abuse with small preimages. We can reconsider in a follow-up though.